### PR TITLE
Configure ODF on nerc-ocp-prod cluster

### LINKED ISSUE #17: Configure ODF on production cluster
We need ODF configured to provide PV and ObjectBucket support to the production cluster.

### PR DESCRIPTION
**Pull request by naved001**
It's ready for review.

- [x] create the appropriate secret in vault
- [x] create fake-metrics app (https://github.com/OCP-on-NERC/nerc-ocp-apps/pull/15)

blocked on argocd being setup to manage nerc-ocp-prod and a cluster-scope app for nerc-ocp-prod.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202467165231356

### CODE REVIEW

File: fake-metrics-server/overlays/nerc-ocp-prod/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: openshift-storage
+resources:
+- ../nerc-ocp-infra
```
Comment (computate) [inline fake-metrics-server/overlays/nerc-ocp-prod/kustomization.yaml:5]: Seems unusual to point the prod overlay to the infra overlay here.

**Comment by larsks** [comment]
This will close https://github.com/OCP-on-NERC/operations/issues/17; please make sure to add an appropriate closes: tag to a commit message in this PR.

**Review by larsks — APPROVED** [review]
This looks fine, but I'd like to hold off merging it until we have the various auth changes (coldfront + github) and the logging stuff sorted out.

**Comment by naved001** [comment]
related PR: https://github.com/OCP-on-NERC/nerc-ocp-apps/pull/15

**Review by computate — APPROVED** [review]

File: fake-metrics-server/overlays/nerc-ocp-prod/kustomization.yaml (hunk 1 above)
Comment (naved001) [reply fake-metrics-server/overlays/nerc-ocp-prod/kustomization.yaml:5]: ideally it should be in some common overlay and then pulled into both environments but as the whole fake metrics thing is a hack to begin with I figured it's probably not necessary to neatly organize it.

**Comment by larsks** [comment]
This depends on https://github.com/OCP-on-NERC/operations/issues/25 and on https://github.com/OCP-on-NERC/operations/issues/26.

**Pull request merged**